<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnKitchen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>111</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>Kitchen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTable5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Table 5</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableServingOrder">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableBellOrder">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOrderOrServe">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>320</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Serve / Order</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbLocation">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Dae-Ta</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbstateLocation">
    <property name="geometry">
     <rect>
      <x>947</x>
      <y>390</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>stateLocation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>390</y>
      <width>68</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>STATUS:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <addaction name="menuHome"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
